cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
```

Comment:
You can make the function private as it won't be used outside the view controller.
Also, I think that you have named it "setupCallbacks()" somewhere else in the code.